README.md

```diff
@@ -83,3 +83,32 @@ for documentation on how to customize branding of the application.
 **NOTE:** Due to an unknown issue, you must execute `./gradlew assembleDebug` one time before opening the project in Android Studio. This will pull in all dependencies that, for whatever reason, are not fetched if Gradle is executed via Android Studio.
 
 After setup is complete, the project can be opened in Android Studio and built as normal.
+
+# Ktlint
+
+SimplyE now has Ktlint enabled. 
+
+**What does this mean?**
+
+This means you no longer get away with committing less than stellar styled code anymore. Anytime you try to commit, your code will be checked for coolness (good style). If your code is cool then nothing different happens, its committed like always. If it is not cool (bad style), then you get an angry *commit failed* message that forces you to correct your style issues before the commit is accepted. Ktlint will provide you with the exact line and directions on exactly how to fix your style issue. Example:
+
+**ktlint**
+```
+simplified-app-simplye/src/main/java/org/nypl/simplified/simplye/SimplyEAccountFallback.kt:20:1: Unexpected indentation (6) (it should be 8) (cannot be auto-corrected)
+```
+
+ **I'll feel bad if my git yells at me for bad style**
+ 
+ Luckily you can always check your own style before committing attempting to commit your code. Simple run the following command from project root:
```

Comment:
Simple -> Simply

:stuck_out_tongue:

Reply:
> Luckily you can always check your own style before committing attempting to commit your code

Can't quite parse that sentence. Think maybe "committing" isn't supposed to be there.